ci/kind/kind-setup.sh

```diff
@@ -354,6 +353,7 @@ networking:
   serviceSubnet: $SERVICE_CIDR
   ipFamily: $IP_FAMILY
   kubeProxyMode: $KUBE_PROXY_MODE
+  dnsSearch: []
```

Comment:
add a comment to indicate it's to prevent inherit search domains from the host which slows down DNS resolution and cause problems to IPv6 only clusters running on IPv4 host.